source/contributing/implementation/pr.rst

```diff
@@ -51,7 +51,7 @@ A few things that can be taken from this:
 
 * Links to any implementation PRs in clear view at the top of the PR, this can be achieved with GitHub Markdown
 
-.. code-block:: none
+.. code-block:: text
```

Comment:
Not sure on this one: Should we use `md` instead?

Reply:
md but you need to change the line below to add spaces around the  | for it to format correctly
![md_markdown](https://user-images.githubusercontent.com/5409394/42483527-0d5a641e-83bc-11e8-8e60-b1840435d866.png)